blog/2023-03-03-solidity-vulnerabilities-signature-replay-attack/solidity-vulnerabilities-signature-replay-attack.md

```diff
@@ -0,0 +1,356 @@
+---
+title: Solidity Vulnerabilities: Signature Replay Attack
```

Comment:
remove the ":" on this line, It's causing deployment error instead how bout something like 
Solidity Vulnerabilities (Signature Replay Attack) or just Signature Replay Attack like in your user.tsx file